project_event/views/project_project_view.xml

```diff
@@ -26,6 +26,12 @@
         <field name="model">project.project</field>
         <field name="arch" type="xml">
             <form string="Project Event">
+            <header>
+                    <button name="action_option" string="Option" attrs="{'invisible': [('state', '!=', 'draft')]}" type="object"/>
+                    <button name="action_accept" string="Accept" attrs="{'invisible': [('state', 'in', ['accepted','canceled'])]}" type="object"/>
+                    <button name="action_cancel" string="Cancel" type="object" attrs="{'invisible': [('state', 'in', ['canceled'])]}"/>
+                    <field name="state" widget="statusbar" statusbar_visible="draft,option,accepted,canceled"/>
+                </header>
```

Comment:
@eilst : Look goods but you are working on the old version of the files. Take  the latest code when you are updating the comments

Reply:
well it depends, if my PR is merged before yours.. still the current version